TodoListClient/tests/acceptance/login-test.js

```diff
@@ -0,0 +1,54 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentURL, click, fillIn, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import {
+  authenticateSession,
+  invalidateSession,
+} from 'ember-simple-auth/test-support';
+
+module('Acceptance | Login', (hooks) => {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('redirects to index if authenticated', async function (assert) {
+    assert.expect(1);
+
+    await authenticateSession();
+    await visit('/login');
+
+    assert.strictEqual(currentURL(), '/s/todo-items', 'the route is correct');
+  });
+
+  test('can submit valid login creds', async function (assert) {
```

Comment:
Please also add a test for popup message on invalid username/password combination.